www/form/inner.js

```diff
@@ -4260,12 +4258,49 @@ define([
                     }
                 }
             }
+            let shiftButtons;
+            let drag;
+            if (editable) {
```

Comment:
Here you're closing an `if (editable) {` to re-open one just after. Why not add your code to the existing `if` block?